packages/databricks-vscode/src/test/e2e/run_job_on_cluster_with_workspace.e2e.ts

```diff
@@ -129,6 +154,72 @@ describe("Run job on cluster with workspace", async function () {
         webView.close();
     });
 
+    it("should run a jupyter notebook as a job on a cluster", async () => {
+        await startSyncIfStopped();
+        await waitForSyncComplete();
+
+        const workbench = await driver.getWorkbench();
+        const editorView = workbench.getEditorView();
+        await editorView.closeAllEditors();
+
+        // open file
+        const input = await workbench.openCommandPrompt();
+        await sleep(200);
```

Comment:
Isn't it making test flaky? What if command prompt appears after 201 ms? Is there or more reliable way to wait for certain element to appear on the screen?